### PR TITLE
[ADD] dead_mans_switch_client

### PR DESCRIPTION
**Pull request by hbrunn**
Only interesting changes are a5bff06 and abc56b8

### CODE REVIEW

File: dead_mans_switch_client/README.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+* certificate pinning would be nice
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/server-tools/issues/new?body=module:%20dead_mans_switch_client%0Aversion:%209.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```
Comment (elicoidal) [inline dead_mans_switch_client/README.rst:50]: You can simplify this part/

File: dead_mans_switch_client/data/ir_cron.xml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data noupdate="1">
+        <record id="cron_client" model="ir.cron">
+            <field name="name">Dead man&apos;s switch client</field>
```
Comment (elicoidal) [inline dead_mans_switch_client/data/ir_cron.xml:5]: is "&apos;" really necessary?

File: dead_mans_switch_client/demo/dead_mans_switch_client_demo.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline dead_mans_switch_client/demo/dead_mans_switch_client_demo.yml:1]: Is this file really necessary?

**Comment by elicoidal** [comment]
:+1:  (small details)

File: dead_mans_switch_client/README.rst (hunk 1 above)
Comment (hbrunn) [inline dead_mans_switch_client/README.rst:50]: what do you mean?

File: dead_mans_switch_client/demo/dead_mans_switch_client_demo.yml (hunk 3 above)
Comment (hbrunn) [inline dead_mans_switch_client/demo/dead_mans_switch_client_demo.yml:1]: yes

File: dead_mans_switch_client/data/ir_cron.xml (hunk 2 above)
Comment (hbrunn) [inline dead_mans_switch_client/data/ir_cron.xml:5]: yes

File: dead_mans_switch_client/README.rst (hunk 1 above)
Comment (elicoidal) [inline dead_mans_switch_client/README.rst:50]: the newest README template doesnot include the last link.

File: dead_mans_switch_client/data/ir_cron.xml (hunk 2 above)
Comment (elicoidal) [inline dead_mans_switch_client/data/ir_cron.xml:5]: my comment was eaten so maybe not understood: is "&apos;" really necessary.
ignore it otherwise

Comment (hbrunn) [inline dead_mans_switch_client/data/ir_cron.xml:5]: ah, this. not necessary, but allowed https://www.w3.org/TR/REC-xml/#sec-predefined-ent

**Comment by lasley** [comment]
👍  runbot error unrelated
